vector/src/main/res/layout/adapter_item_vector_message_room_versioned.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This FrameLayout is useless

Reply:
Nope, it's necessary to have margin rights as ListView LayoutParams doesn't include margins :-)